components/ai_chat/content/browser/ai_chat_tab_helper.h

```diff
@@ -157,6 +166,8 @@ class AIChatTabHelper : public content::WebContentsObserver,
 
   void GetSearchSummarizerKey(GetSearchSummarizerKeyCallback callback) override;
 
+  bool HasOpenLeoPermission() const override;
```

Comment:
`HasContinueConversationPermission`?

Reply:
`HasOpenAIChatPermission`?

Reply:
Renamed to HasOpenAIChatPermission in https://github.com/brave/brave-core/commit/ab4ea07f080379399d945e2357de36394ecf6fc0.